src/components/App.js

```diff
@@ -107,12 +107,12 @@ export default class App extends React.Component {
     this.handleKeyboardHotKeys = this.handleKeyboardHotKeys.bind(this);
   }
 
-  isToggleHotKey(event) {
-    return [' '].includes(event.key);
+  isSpacePressed(event) {
+    return event.key === ' ';
```

Comment:
@ahmadabdolsaheb I have changed the body of the function since as I first used the `K` key as well, but this key doesn't have anything to do with the toggle issue.